SharedContent/cs-winui-template/WinUI3TemplateCs/MainPage.xaml.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Navigation;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace WinUI3TemplateCs
+{
+
+    public sealed partial class MainPage : Page
+    {
+        public static MainPage Current;
+
+        public List<Scenario> Scenarios
+        {
+            get { return this.scenarios; }
```

Comment:
This could be written more succinctly as `get => this.scenarios;`
Or even better, replace the whole property with:

```cs
public List<Scenario> Scenarios => this.scenarios`
```

Or consider removing the field altogether and assigning the property directly:

```cs
public List<Scenario> Scenarios { get; } = new()
{
    new Scenario() { Title = "Scenario 1", ClassName = typeof(Scenario1_ShortName).FullName },
    new Scenario() { Title = "Scenario 2", ClassName = typeof(Scenario2_ShortName).FullName }
};
```

And if the collection is not expected to change at runtime, consider exposing as an `IReadOnlyList<Scenario>` instead.

Reply:
This was probably inherited from the UWP samples template, which predates many of the C# conciseness features people have become accustomed to.

At least in the UWP side, there are rare cases where the list of scenarios is edited on the fly. (E.g., some scenarios are not available when running on Phone or Xbox.)

Reply:
May consider staying with assigning the property in the SampleConfiguration.cs file, since that is the file we would ask sample authors to edit.